frontend/model/contracts/group.js

```diff
@@ -647,6 +743,15 @@ sbp('chelonia/defineContract', {
           // handles proposal pass or fail, will update proposal.status accordingly
           proposals[proposal.data.proposalType][result](state, message)
           Vue.set(proposal, 'dateClosed', meta.createdDate)
+
+          // update 'streaks.noVotes' which records the number of proposals that each member did NOT vote for
+          const votedMembers = Object.keys(proposal.votes)
+          for (const member of getters.groupMembersByUsername) {
+            const memberCurrentStreak = vueFetchInitKV(getters.groupStreaks.noVotes, member, 0)
+            const memberHasVoted = member === meta.username || votedMembers.includes(member)
```

Comment:
Think this line could be simplified to simply `votedMembers.includes(member)`

Reply:
agree